src/poster/MoviePoster.js

```diff
@@ -1,11 +1,14 @@
-import React from 'react'
+import React from 'react';
+import { Link } from 'react-router-dom';
 import './MoviePoster.css'
 
 const MoviePoster = ({id, poster_path, title, displayMovieDetails}) => {
   return (
     <div id={id} className='movie-poster' onClick={() => displayMovieDetails(id)}>
```

Comment:
I think we can get rid of that onClick property now that we have the Link set up?